### DISCUSSION TITLE
How to separate the output for axios and react-query

### DISCUSSION THREAD

**Discussion by taehwanno** [orval-labs/orval#801] [Q&A]
## Question

When applying `client` option as `react-query`, is there a way to create separate files for code that has only axios as a dependency and code that has react-query as a dependency?

## Context

I have multiple microservices and I want to use openapi client code made with orval for communication between microservices as well as web client. But there is a problem that react, react-dom, react-query should exist as npm dependencies in the backend microservices as well.

**Comment by anymaniax** [comment]
At the moment it’s not possible so you need to generate with a react query client and the axios  one separately.
